ui/apps/platform/src/Containers/SystemHealth/DashboardPage.js

```diff
@@ -80,9 +64,9 @@ const SystemHealthDashboardPage = () => {
             </PageSection>
             <PageSection>
                 <Grid hasGutter>
-                    <GridItem span={8} rowSpan={2}>
+                    <GridItem span={12} rowSpan={2}>
                         <Widget
-                            className="sx-2"
```

Comment:
To prevent dark background in classic dark theme until we update to PatternFly or enable PatternFly dark theme.

Without the class, Cluster Health card would look like the 3 Integrations cards in the first picture above.